src/slot_machine.c

```diff
@@ -355,10 +355,10 @@ static IWRAM_DATA struct SpriteFrameImage *gUnknown_03001188[26];
 // Const rom data.
 extern const struct UnkStruct1 *const gUnknown_083ED048[];
 extern const u16 gPalette_83EDE24[];
-extern const u8 gUnknown_083ECD04[][3];
-extern const u8 gUnknown_083ECE3A[];
-extern const u16 gUnknown_083ECE42[];
-extern const u16 gUnknown_083ECE48[];
+extern const u8 LuckyRoundTable[][3];
```

Comment:
All of these global symbols should be prefixed with `g`.  For example, `gLuckyRoundTable`

Reply:
so should anything declared as extern const have the `g` prefix?

Reply:
sorry posted on wrong account. but yeah, does anything with extern const declaration need the `g` prefix?

Reply:
Yes.  All data or global variables should be prefixed with `g`.  Similarly, static is prefixed with `s`.  The big exception to this is script labels, which do not currently have any naming scheme.